cromshell

```diff
@@ -1124,16 +1124,34 @@ function list()
     tmpFile=$( makeTemp )
     cp ${CROMWELL_SUBMISSIONS_FILE} ${tmpFile}
 
-    while read line ; do 
+    UNREACHABLE_SERVER=""
```

Comment:
make these plural, UNREACHABLE_SERVERS and REACHABLE_SERVERS.  

I would also probably make them local variables and lowercase, since they won't persist across operations anyway.

Reply:
done